packages/circuit-ui/components/SelectorGroup/SelectorGroup.stories.tsx

```diff
@@ -70,4 +70,26 @@ export const Multiple = (args: SelectorGroupProps) => {
   );
 };
 
-Multiple.args = { ...baseArgs, multiple: true };
+MultipleSelection.args = { ...baseArgs, multiple: true };
+
+export const MultipleLines = () => {
+  const [value, setValue] = useState<string>('');
+
+  return (
+    <SelectorGroup
+      {...baseArgs}
+      options={[
+        ...baseArgs.options,
+        {
+          children: 'A whole lotta fruits!',
+          value: 'lotsOfFruits',
+          noMargin: true,
```

Comment:
This `prop` isn't necessary here since the SelectorGroup adds `noMargin` to the Selector component by default.

Reply:
Solved [f0bdfd1](https://github.com/sumup-oss/circuit-ui/pull/1651/commits/f0bdfd1f5658cf931758457799f1a1ffa13c024f)